core/page-assets/loader.php

```diff
@@ -56,7 +56,11 @@ public function enable_assets( array $assets_data ) {
 
 		foreach ( $assets_data as $assets_type => $assets_list ) {
 			foreach ( $assets_list as $asset_name ) {
-				$this->assets[ $assets_type ][ $asset_name ]['enabled'] = true;
+				if ( 'scripts' === $assets_type ) {
+					wp_enqueue_script( $asset_name );
```

Comment:
@KingYes - Currently pro assets will not be loaded, see the usage of `assets_loader->add_assets` in the pro, it's adding the assets but not registering them:
Here in your implementation, you enqueue only by the handle, so all assets must be first registered, probably what's best is to change the `add_ssets` method to also register the assets that are being added.
+
Also, change the tests accordingly.